### PR TITLE
404 page

### PR DESCRIPTION
**Pull request by ezhdi**
---
name: "404 page"
about: issue #97 
---

## 📑 Description of the Change

Add 404 page with left menu.

## Verification Process

rebuild docs and check `_build/html/404.html`

### CODE REVIEW

File: docs/404.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.. |Open In Colab3| image:: https://colab.research.google.com/assets/colab-badge.svg
+.. _Open In Colab3: https://colab.research.google.com/github/maks-sh/scikit-uplift/blob/master/notebooks/pipeline_usage_EN.ipynb
+
+.. |Open In Colab4| image:: https://colab.research.google.com/assets/colab-badge.svg
+.. _Open In Colab4: https://colab.research.google.com/github/maks-sh/scikit-uplift/blob/master/notebooks/pipeline_usage_RU.ipynb
+
```
Comment (maks-sh) [inline docs/404.rst:6]: Please, delete these links 🙌

They are not needed.

**Review by maks-sh — CHANGES_REQUESTED** [review]

**Comment by maks-sh** [comment]
Wow!

A very simple and elegant solution 👍

File: docs/404.rst (hunk 1 above)
Comment (ezhdi) [reply docs/404.rst]: Fixed

**Comment by maks-sh** [comment]
I added a picture to make the page more fun 😅

image

Thank you 👍 

Please do not forget to make a submission on the [ods.ai](https://ods.ai/tracks/open-source-soc2021/competitions/scikit-uplift-hack-soc2021) with a link to this PR.

**Review by maks-sh — APPROVED** [review]

**Pull request merged**